src/borg/remote.py

```diff
@@ -452,23 +590,29 @@ def handle_error(error, res):
                             raise
         self.ignore_responses |= set(waiting_for)
 
+    @api(since=(1, 0, 0))
     def check(self, repair=False, save_space=False):
-        return self.call('check', repair, save_space)
+        pass
 
+    @api(since=(1, 0, 0))
     def commit(self, save_space=False):
-        return self.call('commit', save_space)
+        pass
```

Comment:
I guess this could be written as `commit = api(...)`, but it'll be less obvious, especially to IDEs.